frontend/ui/widget/numberpickerwidget.lua

```diff
@@ -110,7 +110,7 @@ function NumberPickerWidget:init()
     }
 
     local empty_space = VerticalSpan:new{
-        width = math.ceil(self.screen_height * 0.01)
+        width = math.ceil(math.max(self.screen_width, self.screen_height) * 0.01)
```

Comment:
May be instead of having this random `0.01 * some screen dimension` (which has no real logic), we could just use `Size.span.horizontal_default` (from frontend/ui/size.lua), or add another value that feels like the previous computed value?
(Which would need to be done in the other places you did similarly a few days ago.)

Reply:
I thought of
https://github.com/koreader/koreader/blob/5b049a5a8232edb5013af7d3509095fecf4b1a0a/frontend/ui/widget/doublespinwidget.lua#L102

Reply:
Go ahead, add a `span.horizontal_large` if `horizontal_default` is too small :)

Reply:
Tested `Size.span.horizontal_default`, very nice.

Reply:
Sorry, dunno why I mentionned using horizontal_* for a VerticalSpan, see my other comment :/